app/views/feeds/_form_auth_tokens.html.erb

```diff
@@ -1,7 +1,19 @@
-<% if feed.tokens.present? %>
-  <%= form.fields_for :feed_tokens, feed.tokens do |fields| %>
-    <%= render "form_auth_token", form: fields %>
+<% feed.tokens.build %>
+
+<%= form.fields_for :feed_tokens do |fields| %>
+  <%= turbo_frame_tag "feed-token-#{fields.object.id}" do %>
+    <% if fields.object.marked_for_destruction? %>
+      <%= fields.hidden_field :_destroy, value: "1" %>
```

Comment:
So I did get the "unsaved controller" registering a deleted thing as a change, which makes the discard button show up etc.  By adding a `data: {unsaved_target: "changed"}` to the element, which triggers the unsaved `changedTargetConnected` method.

But apparently I broke that doing magic stuff on the audio uploader.  So - I'll open a ticket to fix that!  Then can circle back to this page's UX.

Reply:
TMI: the Unsaved controller determines if the form has "changed" by scanning for [the is-changed class](https://github.com/PRX/feeder.prx.org/blob/master/app/javascript/controllers/unsaved_controller.js#L105).

But - even adding `class: "is-saved"` to this hidden destroy field isn't enough.  Because for some reason, I elected to make the controller [ignore visually hidden elements](https://github.com/PRX/feeder.prx.org/blob/master/app/javascript/controllers/unsaved_controller.js#L113).  So I need to go back and figure out why the heck I did that.

Or maybe you could render a 0x0 non-hidden div right above this destroy field?  May be worth a shot.

```html
<div class="is-changed" data-unsaved-target="changed">
```

Reply:
Hey, looks like this worked!  Neat!